src/internal/analytics/utils/parse-count-text.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Extracts the count value from table header/filter text.
+ *
+ * Parses various counter string formats and extracts the relevant numeric value:
+ * - "Items (100)" - Extracts 100 (first number found)
+ * - "1/100" - Extracts 100 (denominator of fraction, representing total count)
+ * - "100+" - Extracts 100 (first number found)
+ * - "1,234 items" - Extracts 1234 (handles comma-separated numbers)
+ */
+export const parseCountValue = (countText: string | undefined): number | undefined => {
+  if (!countText || typeof countText !== 'string') {
+    return undefined;
+  }
+
+  const target = countText.includes('/') ? countText.split('/')[1] : countText;
```

Comment:
The number found at right part of the slash could at the same time contain a comma, right?

Reply:
As discussed. Removed this use case until we get more data.